src/ProjectTemplates/Web.ProjectTemplates/content/RazorPagesWeb-CSharp/Services/DownstreamWebApi.cs

```diff
@@ -0,0 +1,72 @@
+﻿using System.Net;
+using System.Net.Http;
+using System.Threading.Tasks;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Identity.Web;
+
+namespace Company.WebApplication1
+{
+    public interface IDownstreamWebApi
```

Comment:
https://github.com/dotnet/aspnetcore/pull/24065#discussion_r458052024 Is there a specific reason why we need an interface here?